packages/NodeTypeResolver/PHPStan/ParametersAcceptorSelectorVariantsWrapper.php

```diff
@@ -20,16 +20,11 @@ public static function select(
         Scope $scope
     ): ParametersAcceptor {
         $variants = $reflection->getVariants();
-        if ($node instanceof FunctionLike) {
-            return ParametersAcceptorSelector::selectSingle($variants);
-        }
 
-        if ($node->isFirstClassCallable()) {
+        if ($node instanceof CallLike && $node->isFirstClassCallable()) {
             return ParametersAcceptorSelector::selectSingle($variants);
         }
 
-        return count($variants) > 1
-            ? ParametersAcceptorSelector::selectFromArgs($scope, $node->getArgs(), $variants)
-            : ParametersAcceptorSelector::selectSingle($variants);
+        return ParametersAcceptorSelector::selectFromArgs($scope, $node->getArgs(), $variants);
```

Comment:
there is no `getArgs()` method in `FunctionLike`

Reply:
good catch, thx